src/api/public/apidocs-new/paths/staging_project_name_staging_projects_staging_project_name_accept.yaml

```
@@ -0,0 +1,56 @@
+post:
+  summary: Accept a staging project
+  description: |
+    This accepts all staged requests and sets the project state back to 'empty'.
+    Will return in an error if staging project is not acceptable, unless force is given. You can force
+    accept staging projects with building or failing packages, but not with missing reviews
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/staging_project_name.yaml'
+  responses:
+    '200':
+      description: Get the state of a staging project.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            ok:
+              summary: Staged requests are accepted.
+              value:
+                code: ok
+                summary: 'Ok'
```

Is it not enough with:
```suggestion
    '200':
      $ref: '../components/responses/succeeded.yaml'
```